lib/bundle/brew_installer.rb

```diff
@@ -75,6 +75,11 @@ def link_change_state!
       when false
         puts "Unlinking #{@name} formula." if ARGV.verbose?
```

Comment:
Maybe this warrants a `linked?` check or similar (like below)

Reply:
Done.

Reply:
It's not entirely straight forward with the data that's currently stored. `f[:link?]` stores either `true`, `false`, or `nil`.
- `true` indicates it's linked and keg-only.
- `false` indicates it's not linked and not keg-only.
- `nil` indicates its linkage is typical, either (linked and not keg-only) or (not linked and keg only).

Simply whether the formula is linked or not is not explicitly stored by `Bundle::BrewDumper`.

Reply:
@sjackman Could consider extending Homebrew/brew to provide the information here?

Reply:
Never mind. I figured it out. Fixed in 0e0ba74ec025bb5b9e7b82fcfd15c7a312e2b16d